langchain/callbacks/wandb_callback.py

```diff
@@ -0,0 +1,713 @@
+try:
```

Comment:
the imports in this file need to be made more optional, but im happy to help with that